platoon/tests/functional/test_multi_node_worker.py

```diff
@@ -69,12 +69,12 @@ def test_linked_shared(self):
         if self.worker._multinode:
             try:
                 self.worker.shared_arrays[outsize]
-                self.fail("'sout''s size has not been linked yet to a shared buffer")
+                # self.fail("'sout''s size has not been linked yet to a shared buffer")
             except KeyError:
                 pass
             try:
                 self.worker.shared_arrays[insize]
-                self.fail("'sinp''s size has not been linked yet to a shared buffer")
+                # self.fail("'sinp''s size has not been linked yet to a shared buffer")
```

Comment:
I intend that these should fail. Reasonale is that unless `self.worker.shared` or `self.worker.all_reduce` have already been called at least once, `self.worker.shared_arrays` dict should not contain a key corresponding to this array's size. I realized that it actually fails in this test suite, because `self.worker` has already been used by previous `test_*` test cases...

Please uncomment these 2 (lines 72 and 77) and change line 62 instead to a different size (refer to the comment I made on that line).